tests/integration/version.test.js

```diff
@@ -10,19 +10,48 @@ describe('NODE ENV', () => {
 describe('VERSION TEST', () => {
   beforeEach(() => {
     process.env.API_VERSION = '1.2.3';
+    process.env.MOBILE_VERSION = ['1.2.0'];
   });
   afterEach(() => {
     process.env.API_VERSION = '';
+    process.env.MOBILE_VERSION = '';
   });
 
   describe('POST /version/check-update', () => {
-    it('should check if user should update app', async () => {
+    it('should return that user does not have to update app (old version)', async () => {
       const response = await app.inject({
         method: 'GET',
         url: '/version/check-update?apiVersion=1',
       });
       expect(response.statusCode).toBe(200);
       expect(response.result.data.mustUpdate).toBeFalsy();
     });
+
+    it('should return that user have to update app (old version)', async () => {
```

Comment:
en renommant la methode tu peux simplifier tes noms de tests en should return true ... / should return false

Reply:
idem pour les autres

Reply:
done